ssl/methods.c

```diff
@@ -125,6 +125,12 @@ IMPLEMENT_dtls1_meth_func(DTLS1_2_VERSION, 0, SSL_OP_NO_DTLSv1_2,
                           ossl_statem_accept,
                           ossl_statem_connect, DTLSv1_2_enc_data)
 #endif
+#ifndef OPENSSL_NO_DTLS1_3_METHOD
```

Comment:
This guard makes no sense because we will not have a public DTLSv1_3_method() function. We don't have this for the TLSv1.3 equivalent.

Reply:
Cool will fix it.